scripts/install/install_bmi.sh

```diff
@@ -0,0 +1,58 @@
+set -x
+
+### START BMI setup
+
+TOP_DIR=$(cd $(dirname "$0") && pwd)
+
+# Move to main ims directory for install from script location
+pushd $TOP_DIR/../../
+
+### Install BMI
+
+user=`whoami`
+sudo mkdir -p /var/log/bmi/logs /etc/bmi /opt/bmi /var/lib/tftpboot
+sudo chown -R $user:$user /etc/bmi /var/log/bmi /opt/bmi /var/lib/tftpboot /etc/tgt
```

Comment:
You made a directory called /var/log/bmi/logs as root and configured BMI to dump logs in it.
You are chowning /var/log/bmi which will not allow BMI to write to the /logs subfolder as it will be owned by root and BMI will run as $user.